### PR TITLE
Auth subject place access points, refs #12593

### PR DESCRIPTION
**Pull request by sbreker**
This is Feature 1 of a larger project.

### CODE REVIEW

File: lib/model/QubitActor.php [hunk 1]
```diff
@@ -607,4 +607,20 @@ public function getOccupations()
 
     return QubitObjectTermRelation::get($criteria);
   }
+
+  public function getSubjectAccessPoints()
+  {
+    return $this->getTermRelations(QubitTaxonomy::SUBJECT_ID);
+  }
+
+  public function getPlaceAccessPoints(/*array $options = array('events' => false)*/)
```
Comment (mcantelon) [inline lib/model/QubitActor.php]: Ghost code. ;)

File: lib/model/QubitObjectTermRelation.php [hunk 2]
```diff
@@ -70,4 +70,40 @@ public static function getOneByObjectId($objectId)
...
     return QubitObjectTermRelation::getOne($c);
   }
+
+  public static function setTermRelationByName($name, $options)
+  {
+    // see if subject term already exists
+    $criteria = new Criteria;
+    $criteria->addJoin(QubitTerm::ID, QubitTermI18n::ID);
+    $criteria->add(QubitTerm::TAXONOMY_ID, $options['taxonomyId']);
+    $criteria->add(QubitTermI18n::NAME, $name);
```
Comment (mcantelon) [inline lib/model/QubitObjectTermRelation.php]: Do we need to worry about culture?

**Review by mcantelon — CHANGES_REQUESTED** [review]
Looks good! One one minor thing found and a question.

File: lib/model/QubitActor.php (hunk 1 above)
Comment (sbreker) [reply lib/model/QubitActor.php]: Removed this - see latest commit.

File: lib/model/QubitObjectTermRelation.php (hunk 2 above)
Comment (sbreker) [reply lib/model/QubitObjectTermRelation.php]: I think you're right.  I have added a new commit that adds 'culture' to $options and updates the calls accordingly.

**Comment by mcantelon** [comment]
Looks good! Nice work! 👍

**Review by mcantelon — APPROVED** [review]

**Pull request closed without merging**